ste/mgr-JobMgr.go

```diff
@@ -744,6 +841,26 @@ func (jm *jobMgr) QueueJobParts(jpm IJobPartMgr) {
 	jm.coordinatorChannels.partsChannel <- jpm
 }
 
+// deleteJobPartsMgrs remove jobPartMgrs from jobPartToJobPartMgr kv.
+func (jm *jobMgr) deleteJobPartsMgrs() {
+	jm.Log(pipeline.LogInfo, "JobPartsMgrsDelete enter")
```

Comment:
Nit: It is always good practice to use ClassName.FunctionName for logging statements. `JobMgr::deleteJobPartsMgr`. Debugging becomes easier and logs become more meaningful.